fraim/workflows/code/workflow.py

```diff
@@ -157,7 +157,7 @@ async def triage_with_semaphore(vuln: sarif.Result) -> sarif.Result | None:
             logger.debug("Filtering the triaged vulnerabilities by confidence")
             high_confidence_triaged_vulns = filter_results_by_confidence(triaged_vulns, self.args.confidence)
 
-            self.history.append_record(
+            history.append_record(
```

Comment:
I actually have this fixed on a branch as well. This same log is duplicated in `process_chunk_with_semaphore` so I just removed this line entirely.

Reply:
Nice, thanks.  Just checked again. I must have discovered that duplicate too, because I'd removed it on my triager working branch.  I changed this PR to just remove the line, as you did.